### PR TITLE
Changed 'new note' to button

### PR DESCRIPTION
**Pull request by andreasjacobsen93**
As proposed here https://github.com/nextcloud/notes/issues/44, I changed the 'new note' link to a button.

See https://github.com/nextcloud/notes/pull/103 for the previous PR
Had to make a new PR because of a problem with my old fork.

capture
Now the button is similar to the one in Contacts.

### CODE REVIEW

File: css/notes.css [hunk 1]
```diff
@@ -5,18 +5,27 @@
...
-    padding-left: 2px;
+#app-navigation #note-add #new-note-button {
+	margin: 14px auto;
+	width: calc(100% - 20px) !important;
+	text-align: left;
+	padding-left: 34px;
+	background-position: 10px center;
 }
-
+
```
Comment (Henni) [inline css/notes.css]: unnecessary whitespace

**Comment by Henni** [comment]
regarding your fork:
maybe @jancborchardt can add you to the nextcloud organization. Then you could work directly on branches inside this repo.

**Comment by andreasjacobsen93** [comment]
Removed whitespace 👍

**Comment by jancborchardt** [comment]
Yup, @andreasjacobsen93 you are in the Nextcloud organization. For easier collaboration you now don't need personal forks of the repositories anymore but can directly create branches in the nain repositories and open pull requests from there. :)
(Don't worry about doing sonething wrong: Important branches like master and stable are protected and can only be modified via reviewed pull requests. :)
